notifier/notifier.go

```diff
@@ -0,0 +1,30 @@
+package notifier
+
+import (
+	v1 "k8s.io/api/core/v1"
+)
+
+type Notifier interface {
+	NotifyPodTermination(pod v1.Pod) error
+}
+
+type Notifiers struct {
+	notifiers []Notifier
+}
+
+func New() *Notifiers {
+	return &Notifiers{notifiers: make([]Notifier, 0)}
+}
+
+func (m *Notifiers) NotifyPodTermination(pod v1.Pod) error {
+	for _, n := range m.notifiers {
+		if err := n.NotifyPodTermination(pod); err != nil {
+			return err
```

Comment:
We could think about not exiting early here.

Imagine we have two notifiers, one for slack and one for email. When the slack notifier fails we could still try to trigger the email notifier. We could collect the errors via [multierror](https://github.com/linki/chaoskube/pull/154/files#diff-634e8348cbd5b2d6a35703a1224c7b51R177).

Reply:
Totally agree with this one, I'll work on that !

Reply:
Implemented and added some tests for the new behaviour

Reply:
That's awesome. Thanks a lot!